components/SwipeRow.js

```diff
@@ -584,6 +586,14 @@ SwipeRow.propTypes = {
      * Tension for the open / close animation
      */
     tension: PropTypes.number,
+	/**
```

Comment:
Can you please run `yarn run fix` on the changes, I think indentation is off here

Reply:
Added commit after yarn run fix.